fabtests/prov/lpp/src/ipc.c

```diff
@@ -233,7 +234,7 @@ static void *info_server_thread(void *arg)
 
 	int info_server_sock = setup_server(info_server_port);
 
-	while (1) {
+	while (server_running) {
```

Comment:
`sever_running` is set to 1 before the thread is created and never set to 0. What's the difference from the old code?

Reply:
This is just to stop the coverity complaining about dead code after the while loop.